### DISCUSSION TITLE
Ajouter l'historique du projet

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2651] [Idées]
Je suis tombé sur la page de présentation du projet Archifiltre, et j'aime bien le concept de timeline du projet. J'en avais entendu parlé il y a longtemps et ça m'a permis de savoir ce qu'ils avaient fait entre temps.

Ça pourrait être chouette d'avoir la même chose pour mon-entreprise.fr 

[![Screenshot_2021-06-03 Qui sommes-nous ](https://user-images.githubusercontent.com/1775934/120665811-655d2200-c48c-11eb-8e56-729cd603dd3e.png)](https://archifiltre.fabrique.social.gouv.fr/qui-sommes-nous)

**Comment by mquandalle** [comment]
Autre exemple : https://www.fabrique.social.gouv.fr/startups/code-du-travail-numerique/

**Comment by mquandalle** [comment]
Peut-être qu'on pourrait intégrer des événements personnalisés sur https://dashlord.incubateur.net/url/mon-entreprise-urssaf-fr/informations/ ? Et éventuellement avoir une iframe Dashlord qu'on puisse intégrer sur le site ? cc @revolunet

![image](https://user-images.githubusercontent.com/1730702/191685333-e535babe-50fd-4a37-bc5e-09ae741272bf.png)
